### PR TITLE
Sus issue 1604

### PR DESCRIPTION
**Pull request by Duende13**
Variants are displayed Alphabetically and  Ascending unit_value. #1604 
Let me know your thoughts @lin-d-hop

### CODE REVIEW

File: app/serializers/api/enterprise_serializer.rb [hunk 1]
```diff
@@ -141,11 +141,11 @@ def active
...
-      :hub => "/assets/map_005-hub.svg",
-      :hub_profile => "/assets/map_006-hub-profile.svg",
-      :producer_hub => "/assets/map_005-hub.svg",
-      :producer_shop => "/assets/map_003-producer-shop.svg",
-      :producer => "/assets/map_001-producer-only.svg",
+      :hub => "/assets/map_005-hub.png",
+      :hub_profile => "/assets/map_006-hub-profile.png",
+      :producer_hub => "/assets/map_005-hub.png",
+      :producer_shop => "/assets/map_003-producer-shop.png",
+      :producer => "/assets/map_001-producer-only.png",
```
Comment (enricostano) [inline app/serializers/api/enterprise_serializer.rb]: Is this related to variant order? If not this should go into another PR IMHO

**Comment by lin-d-hop** [comment]
Hi @Duende13 
If you look at the commit history here you can see you also have the commits from your last PR included.
Looks like you carried them over to the new branch when you created it. In future you can reset your branch to upstream master before working on it. And if you need to update your branch mid PR make sure you rebase it so the merge commits don't get included in the new PR.

I created a new PR with just the commit in question for this issue #1625 

Going to give it a whirl on staging now :-)

Thanks for your work on this!!
